app.rb

```diff
@@ -0,0 +1,179 @@
+require_relative './person'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './book'
+
+class App
+  attr_accessor :books, :persons, :rentals
+
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def list_books
+    puts '**Book List**'
+    books.each_with_index do |book, index|
+      puts "[#{index + 1}] Title: #{book.title} by #{book.author}"
+    end
+  end
+
+  def list_people
+    puts 'All people'
+    persons.each_with_index do |person, index|
+      puts "[#{index + 1}] ID: #{person.id} Name: #{person.name} Age: #{person.age}"
+    end
+  end
+
+  def create_teacher
+    puts 'Creating teacher...'
+    name = '', age = 0, specialization = ''
+    loop do
+      print 'Name: '
+      name = gets.chomp
+      print "\n"
+      print 'Age: '
+      age = gets.chomp
+      print "\n"
+      print 'specialization: '
+      specialization = gets.chomp
+      print "\n"
+      age = age.to_i
+      break if name.length.positive? && specialization.length.positive? && age.positive?
+
```

Comment:
- [ ] I am not able to create a `teacher` when I try to add `specialization` I get an error please address that
![gg](https://user-images.githubusercontent.com/44745855/181918829-bfcc40a4-a4a5-419f-a8f6-d17a1bea2dd6.png)

Reply:
@DeliceLydia  addressed the issue 👍🏽 

![image](https://user-images.githubusercontent.com/94127418/181925447-81873149-f68b-4050-8336-5c2c51e17f2d.png)